sdk/app_cpu1/common/drv/timing_manager.c

```diff
@@ -441,6 +441,40 @@ double timing_manager_get_time_per_sensor(sensor_e sensor)
     return time;
 }
 
+/*
+ * Get the time since the sensor value was gathered for the requested sensor, in microseconds
+ */
+double timing_manager_get_time_since_sensor_poll(sensor_e sensor)
```

Comment:
Rename this function: ~`timing_manager_get_time_since_sensor_poll()`~  --> `timing_manager_get_time_since_sensor_done()`